src/withPermutation.tsx

```diff
@@ -54,7 +54,7 @@ const Wrapper = styled.div<{
     width: 100%;
   }
   table#permutation-table .permutation-inner-table td,
-  th {
+  table#permutation-table th {
```

Comment:
To solve the issue with global styles, we've attached the id `permutation-table` to each table to prevent the styling of permutation-table from being applied to Stories that use the table element. However, this rule was excluded for `<th>` items.

After adding the rule and checking it, the styling was no longer injected into the Story.

GLFM 🤗